pontoon/administration/templates/admin_project.html

```diff
@@ -277,6 +277,11 @@ <h3>
           <label for="id_disabled">
               {{ form.disabled }}{{ form.disabled.label }}
           </label>
+      </div>
+       <div class="checkbox clearfix">
+          <label for="id_is_dummy">
+              {{ form.is_dummy }}{{ form.is_dummy.label }}
+          </label>
```

Comment:
I don't think creating dummy projects is a thing that will happen more than once. Thus I think it would be better to not expose this option in the admin panel, and to create the one dummy project we're going to use as a data migration. That makes sure that anyone using Pontoon will have it, which is good because it will be a requirement for the website to work properly.

Reply:
Yeah, I just added it for my ease. Will surely remove it eventually.